### ISSUE TITLE
Unable to reproduce

### ISSUE THREAD

**Issue by Jie-Yuan** [microsoft/LightGBM#2444]
I set random_state=666:

LGBMClassifier(boosting_type='gbdt', class_weight=None, colsample_bytree=0.617,
               importance_type='split', learning_rate=0.005, max_depth=-1,
               metric='auc', min_child_samples=31, min_child_weight=0.01,
               min_split_gain=0.884, n_estimators=30000, n_jobs=-1,
               num_leaves=16, objective='binary', random_state=666,
               reg_alpha=0.631, reg_lambda=0.81, scale_pos_weight=1,
               silent=True, subsample=0.788, subsample_for_bin=200000,
               subsample_freq=8, verbosity=-1)

**Comment by Jie-Yuan** [comment]
sklearn api

**Comment by guolinke** [comment]
could you describe your problem with more details?

**Comment by StrikerRUS** [comment]
@Jie-Yuan Please follow [my comment in #2448](https://github.com/microsoft/LightGBM/issues/2448#issuecomment-535056081) and post your feedback there to not split the discussion of the same issue into multiple topics.
